store/src/java/com/zimbra/cs/filter/ZimbraMailAdapter.java

```diff
@@ -959,4 +965,12 @@ public boolean isCapable(String capability) {
     public boolean isImplicitKeep () {
         return fieldImplicitKeep;
     }
+
+    public PARSESTATUS getEheParseStatus() {
+        return eheParseStatus;
+    }
+
+    public void setEheParseStatus(PARSESTATUS eheParseStatus) {
```

Comment:
Okay.  I'll change it to setEditHeaderParseStatus and getEditHeaderParseStatus respectively.